umka/font.um

```diff
@@ -14,43 +14,51 @@ type Filter* = enum {
 }
 //~~
 
-//~~opaque Font
-type Font* = struct { _: ^struct{} }
-//~~
+// TODO: should this be exported and documented?
+type TtfFont* = struct { _: ^struct{} }
```

Comment:
I think it's OK to export it. This is just a minor nitpick - I think `TtfFont` doesn't look very nice, maybe we can call it `Truetype`.